docs/docs/addons-shallow-compare.md

```diff
@@ -1,19 +1,30 @@
 ---
 id: shallow-compare
 title: Shallow Compare
-permalink: docs-old/shallow-compare.html
-prev: perf.html
-next: advanced-performance.html
+permalink: docs/shallow-compare.html
+layout: docs
+category: Reference
+prev: create-fragment.html
+next: test-utils.html
 ---
 
```

Comment:
Can you please add a note here that `shallowCompare` is considered legacy (not deprecated), and that we also recommend `PureComponent` instead?